src/standard/http/response/StatusByCode.ts

```diff
@@ -0,0 +1,282 @@
+    Description: StatusDescription.Imateapot,
+  },
+  [StatusCode.MisdirectedRequest]: {
+    Code: StatusCode.MisdirectedRequest,
+    Description: StatusDescription.MisdirectedRequest,
+  },
+  [StatusCode.UnprocessableEntity]: {
+    Code: StatusCode.UnprocessableEntity,
+    Description: StatusDescription.UnprocessableEntity,
+  },
+  [StatusCode.Locked]: {
+    Code: StatusCode.Locked,
+    Description: StatusDescription.Locked,
+  },
+  [StatusCode.FailedDependency]: {
+    Code: StatusCode.FailedDependency,
+    Description: StatusDescription.FailedDependency,
+  },
+  [StatusCode.TooEarly]: {
+    Code: StatusCode.TooEarly,
+    Description: StatusDescription.TooEarly,
+  },
+  [StatusCode.UpgradeRequired]: {
+    Code: StatusCode.UpgradeRequired,
+    Description: StatusDescription.UpgradeRequired,
+  },
+  [StatusCode.PreconditionRequired]: {
+    Code: StatusCode.PreconditionRequired,
+    Description: StatusDescription.PreconditionRequired,
+  },
+  [StatusCode.TooManyRequests]: {
+    Code: StatusCode.TooManyRequests,
+    Description: StatusDescription.TooManyRequests,
+  },
+  [StatusCode.RequestHeaderFieldsTooLarge]: {
+    Code: StatusCode.RequestHeaderFieldsTooLarge,
+    Description: StatusDescription.RequestHeaderFieldsTooLarge,
+  },
+  [StatusCode.UnavailableForLegalReasons]: {
+    Code: StatusCode.UnavailableForLegalReasons,
+    Description: StatusDescription.UnavailableForLegalReasons,
+  },
+  [StatusCode.InternalServerError]: {
+    Code: StatusCode.InternalServerError,
+    Description: StatusDescription.InternalServerError,
+  },
+  [StatusCode.NotImplemented]: {
+    Code: StatusCode.NotImplemented,
+    Description: StatusDescription.NotImplemented,
+  },
+  [StatusCode.BadGateway]: {
+    Code: StatusCode.BadGateway,
+    Description: StatusDescription.BadGateway,
+  },
+  [StatusCode.ServiceUnavailable]: {
+    Code: StatusCode.ServiceUnavailable,
+    Description: StatusDescription.ServiceUnavailable,
+  },
+  [StatusCode.GatewayTimeout]: {
+    Code: StatusCode.GatewayTimeout,
+    Description: StatusDescription.GatewayTimeout,
+  },
+  [StatusCode.HTTPVersionNotSupported]: {
+    Code: StatusCode.HTTPVersionNotSupported,
+    Description: StatusDescription.HTTPVersionNotSupported,
+  },
+  [StatusCode.VariantAlsoNegotiates]: {
+    Code: StatusCode.VariantAlsoNegotiates,
+    Description: StatusDescription.VariantAlsoNegotiates,
+  },
+  [StatusCode.InsufficientStorage]: {
+    Code: StatusCode.InsufficientStorage,
+    Description: StatusDescription.InsufficientStorage,
+  },
+  [StatusCode.LoopDetected]: {
+    Code: StatusCode.LoopDetected,
+    Description: StatusDescription.LoopDetected,
+  },
+  [StatusCode.NotExtended]: {
+    Code: StatusCode.NotExtended,
+    Description: StatusDescription.NotExtended,
+  },
+  [StatusCode.NetworkAuthenticationRequired]: {
+    Code: StatusCode.NetworkAuthenticationRequired,
+    Description: StatusDescription.NetworkAuthenticationRequired,
+  },
+};
```

Comment:
This could be evaluated not built. We can iterate through the entries of `StatusCode` and then just assign to the description. Assuming we are not using TS enums since they create some weird behaviour.

Reply:
agreed. and those enum resources you posted were hella helpful. also, this whole response status code enum/map thing originally started in core and i put it further out into standard cause it made sense. however, exercising the code and using this implementation felt freaking weird.

wondering two things about this:
1. do you know a better way to structure this? i remember you making the MIME DB stuff we had better. even node's and deno's impl of response status code dictionaries are kind of weird. like they gave it a shot and said meh, whatever, let's roll with it.
2. if we keep a response status code dictionary, should we just stick to numbers (eg, 500) instead of nice looking identifiers (eg, InternalServerError) and leave response status code enums/maps/dictionaries/etc to user land?

Reply:
So I think we should use the enums concept (with objects like I explained) and allow users to use them. But we can use them throughout our codebase.

https://www.typescriptlang.org/play?#code/MYewdgzgLgBAylAhlArhA0gUwJ4RgXhgG0AiAeXRIBoYSBhAJQFEBBAFSYBFraAhFzgH1mARQCqTOGx4kAcmTaCAYmTGzuAXRiI8oSFABQe6PCSoIdEABNMBGAG8DMGBQBcMAEwAGL1ScxGVg5Od28ARj9nfiFRCSl3ABYfSJh5RRU1EJgkhL8AX21dcGgAbgMobAAHWwRkNEsbOwAKCuqQADNTOotrTABKIgBrHA6YVsxR2vMGzA0jYtgptE5MCGAAJwBLSqhN8DtHZzdaMkHqf0D2Lnd6dcxkTCtzqIFhJnFJNhveRCsYBkwAEcUKsoM9UgplKp1DdZCBYEoQCgwE98oUYMYoGVxl1zCs1lsdnswM1xpMzMtVhttrtwANhthRmTOksIPjqUTwHN5voYHcIJVipgwnYwJgAO7-VaCyCYJpgFAAG0VNHs0G67lZMwAdBQaOrzGxMAAPKCailsqmE2lgXXoGB5PoGAwAehdMAAkp1xbZFSAQIMMX6IJhFdgaGB4QALTZgADmMEqdwAbpgwFA8GgYO11iAALYwKwgWNxnkmfkykMeUUSqUCoXypUqhwGtDm7o6vUwVsQI2m9t4q004na6JvD5SB1O13ugDqtmAiBJ7RA62Atk2sHFm6j2looDzguRTxgaZQebLiwtB38xyIWt6dporPZ1pHFA0KUuwXc94tOu-LhnwtV9h3AbVAM4T9-DHWJPl-B8bFHV44KkYDulAzlbVg944jYaDnDSKFMgQ-9HyIjJ1HQwcCTA20KOhKC0R0DEFnKKoamvQgWg48lunpEZOmZXE0A0MoZ1SEBJSgKNbDQTB1l0JcYHksZZJEnpGlXDTMJtGBYysTZk02KwUEQZVsBgbTFxJAAjWwdAgFA7j+GScG0O4VIgEs1IXVc7mAWBKkQTZ1hgbcZI0y9iFAGwaCgE0oC0QhWTtMpMT5aUhQAZhrSUAXrWVG2VVUe3cWLMH1C0+zNMZEqnZ03RgWcdzUzY8GgXN4zGKofPABdlMS9ZEEC3zWPWALYGTcyQTwEgrGwMBEDzTZFwskhooZEVCHIShopXEBto0oYcDCbkMoZCAjtISCZBwidpC0FjMQMFcwqaC73NGS6wj6Bx-Ayg7qxSi0TuwOY8kaudbDzNBYBsyN4buB4xoAA1ZVHTwVAt9nWZFdjzWxowU7cQ3C30QDJtyesqHylz+TcYHsg9VmIbH7PWKqtnjMTotZBh8c2Qm7DIWyACtMEC7U0ygLZViaRD+m1FyUHXJp2mRQLiRgJok0wZMaDBmhpsVEEND+w5ExTMHkpgE2QTKZw7lQdYST15MyjyVUChYgFQHWKwAB5OpLCNzw5gA+PoSiAA

Reply:
done in [2409f9d](https://github.com/drashland/drash/pull/698/commits/2409f9d02fd28fcb7b56975ceda1a85cfb2be0b1)